### ISSUE TITLE
unable to use mod in general

### ISSUE THREAD

**Issue by TheKing8900** [paulevsGitch/BCLib#83]
I recently tried enabling better nether and better end together and got directed to download BClib, and so I did. The problem now is that; even after disabling my only other mod in the mod pack (because at first, I thought it may be a backwards compatibility error), it seems like BCLib is making my game unplayable by an error when I try to start up my game. Any time I attempt to start up the software with BClib installed into my modpacks folder the game simply will not open. I know this because it won't open while it's in my active mods folder but it will open once BClib and the better mods are in my disabled mods folder.

TL;DR BClib makes it so I can't even open up my game as it gives me an error every time I try to open up my Fabric API version of Minecraft, and once it is disabled I can play with no problems

P.S. Yes, I did try reinstalling and the version is the newest one available as of MC 1.18.1 on February 2, 2022

**Comment by JDtheHutt** [comment]
Please post your most recent log and crash log in pastebin and link here so that details can be seen.  There is an issue currently involving TerraBlender, but exact mods installed and their versions plus any error messages would identify if this is the same or something else.

**Comment by TheKing8900** [comment]
https://pastebin.com/CKZfjura is the best I can manage, unsure if this contains all the necessary information

**Comment by paulevsGitch** [comment]
You are loading mod for 1.17 on 1.18, please use correct mod version
